core/assets/bundles/bundle.properties

```
@@ -776,7 +776,7 @@ bar.corereq = Core Base Required
 bar.drillspeed = Drill Speed: {0}/s
 bar.pumpspeed = Pump Speed: {0}/s
 bar.efficiency = Efficiency: {0}%
-bar.boost = Boost: {0}%
+bar.boost = Boost: x{0}%
```

an alternative might be including the "default" boost and indicating the increase/change somehow, perhaps like this:

```suggestion
bar.boost = Boost: 100% -> {0}%
```

---

No, if power is not enough, it will display as 100% -> 80% etc.

---

A way is using "boost: +50%" instead of 150%

---

Maybe there's something needed to change in 
Mindustry/core/src/mindustry/world/blocks/defense/OverdriveProjector.java /
like when the power is low, show 'low power' instead of '80%', and if it is enough, show '+50%'.